[ios] Match Spedcam strings with Android, fix `never` and `stop` strings issues (#7582)
organicmaps/organicmaps: data/strings/strings.txt

  [pref_tts_speedcams_auto]
    comment = Speed camera settings menu option - Warn (about speedcams) if risk of exceeding speed limit
    tags = android,ios

>     en = Warn if there is a risk of exceeding the speed limit

Reviewer: Sorry, I was sure that we fixed it.
en = Warn when the speed limit is exceeded

>     en = Warn if there is a risk of exceeding the speed limit

Reviewer: ```suggestion
    en = Warn when the speed limit is exceeded
```

Reviewer: Let's do it in a separate PR.
